### DISCUSSION TITLE
RAM 4.0 Statusupdate

### DISCUSSION THREAD

**Discussion by ssteinbuss** [International-Data-Spaces-Association/IDS-RAM_4_0#109] [Announcements]
What is the current status of IDS-RAM 4.0?

**Comment by HeinrichPet** [comment]
**Process Layer**
- [x] Definition of work & Responsibilities - Finshed [Components](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/discussions/62) & [Onboarding section](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/issues/59)
- [ ]  Data Offering/Metadata Broker - Draft nearly finished: PR #76 & PR #100 
- [ ] Data Exchange - PR #113 
- [ ]  Publishing and using Data Apps - Waiting for draft
- [ ]  Clearing House - Waiting for draft
- [ ]  Vocabulary Hub - Waiting for draft
- [ ]  DAPS - [Partly included](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/main/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_1_Onboarding.md) / Needs to be reviewed in detail and extended if necessary
- [ ]  ParIS - Waiting for draft
- [ ] Contract Negotiation - PR #111 
- [ ]  Policy Enforcement - Waiting for draft

Jan. 25 assessment of whether the draft will be ready by Feb. 28: Yes.

**Comment by HeinrichPet** [comment]
**System Layer**
- [x] Definition of work & Responsibilities - [Finished](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/discussions/62)
- [ ] Connector - PR #110 
- [ ] Metadata Broker - PR #79 
- [ ] ParIS - PR #79 
- [ ] Vocabulary Hub - PR #79 
- [ ] DAPS/Identity Provider -  Waiting for draft
- [ ] App Store - Waiting for draft
- [x] Clearing House - Draft finished PR #103 

[Policy Enforcement?](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/discussions/62#discussioncomment-1814314)

     

Jan. 25 assessment of whether the draft will be ready by Feb. 28: Yes.

**Comment by ssteinbuss** [comment]
### Introduction
The Introduction does not require a major rework. We will update the section during the last steps of the RAM update to align it with the content. Nevertheless, please review and provide specific changes that should be adopted.

**Comment by ssteinbuss** [comment]
### Context of the International Data Space

- [x]  2.1 Data in the Smart Service Welt

- [ ]  2.2 Data Sovereignty as a Key Capability

- [ ]  2.3 Data as an Economic Good

- [ ]  2.4 Data Exchange and Data Sharing

- [ ]  2.5 Industrial Cloud Platforms

- [ ]  2.6 Big Data and Artificial Intelligence

- [ ]  2.7 The Internet of Things and the Industrial Internet of Things

- [ ]  2.8 Blockchain

- [ ]  2.9 Towards legal interoperability: federated frameworks for data sharing agreements and terms-of-use

- [ ]  2.10 General Data Protection Regulation

- [ ]  2.11 Contribution of the International Data Space to Industry 4.0 and the Data Economy

- [ ]  new section on data privacy

- [ ]  Update UC Content

**Comment by mokamhuber** [comment]
Certification Perspective (https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/projects/13):

- [x] 4.2. Introduction #89 
- [x] 4.2.1. certification aspects on different RAM layers #124 
- [x] 4.2.2. Roles #90 
- [x] 4.2.3. Operational Environment Certification #93 
- [x] 4.2.4. Component Certification #125 
- [x] 4.2.5.1. Processes: Approval of Evaluation Facilities #91 
- [ ] 4.2.5.2. Certification Process #126

**Reply by mokamhuber** [reply]
Status update: all sections have been revised and are currently in review

**Comment by mokamhuber** [comment]
Security Perspective (https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/projects/11):

- [ ] 4.1 Security Perspective #68 
- [x] 4.1.1. Security Aspects addressed by the different layers of the IDS-RAM #80 
- [ ] 4.1.2. Identity and Trust Management #69 
- [ ] 4.1.3. Securing the Platform #70 
- [ ] 4.1.4. Securing Applications #72 
- [x] 4.1.5. Securing Interaction with Other IDS Components #73 
- [ ] 4.1.6. Usage Control #71 und https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/projects/12
